Source/JavaScriptCore/assembler/ARM64Assembler.h

```diff
@@ -3744,7 +3756,7 @@ class ARM64Assembler {
 
     static JumpLinkType computeJumpType(LinkRecord& record, const uint8_t* from, const uint8_t* to)
     {
-        JumpLinkType linkType = computeJumpType(record.type(), from, to);
+        JumpLinkType linkType = computeJumpType(record, record.type(), from, to);
```

Comment:
Do we still need to pass `record.type()` if we're already passing `record`?  `computeJumpType` should always use the `type()` from the `record`, right? 

I see that you had 2 `computeJumpType` methods to begin with.  I don't see a caller to the other form.  Should the 2 methods be consolidated?

Reply:
Changed.